exercises/raindrops/uRaindropsTest.pas

```diff
@@ -0,0 +1,179 @@
+unit uRaindropsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  RaindropsTest = class(TObject)
+  public
+    [Test]
+    procedure the_sound_for_1_is_1;
+
+    [Test]
+    [Ignore('Comment these lines to run these tests')]
```

Comment:
We do it in the Ruby track, see [hamming test](https://github.com/exercism/xruby/blob/master/exercises/hamming/hamming_test.rb) for example.